_episodes/09-plotting.md

```diff
@@ -36,21 +36,27 @@ plt.xlabel('Time (hr)')
 plt.ylabel('Position (km)')
 ~~~
 {: .python}
```

Comment:
you need to insert a blank line after `{: .python}`. The language tags should also be changed to `{: .language-python}` to allow for syntax highlighting.